doc/harmony-smp_installation_guide.md

```diff
@@ -84,10 +84,10 @@ See eDelivery documentation \[[TERMS](#Ref_TERMS)\].
 
 ### 2.1 Prerequisites to Installation
 
-The SMP is officially supported on the following platforms:
+The SMP is officially supported on the following operating system:
```

Comment:
This should be plural and I would also mention the supported architectures => "operating systems (`x86-64`, `arm64`)".